### PR TITLE
pixel-perfect breakpoint icons

### PR DESCRIPTION
**Pull request by jlongster**
Breakpoints before looked a little off and it was driving me crazy. This makes them align perfectly with the line numbers, and also load instantly. 

On my computer, I'm still having the issue where line numbers are slightly off, so the breakpoints are 1px off for me but they should be perfectly aligned for everyone else. I still need to figure out the line number issue that I'm seeing.

### CODE REVIEW

File: public/js/components/Editor.js [hunk 1]
```diff
@@ -77,6 +81,13 @@ const Editor = React.createClass({
     }
   },
 
+  resizeBreakpointGutter() {
```
Comment (jlongster) [inline public/js/components/Editor.js]: The technique here anchors the breakpoint to the beginning of the line, or the "right side" of the line number gutter. That way if there are 10,000,000 lines and the gutter expands really large, the breakpoint will be anchored to the right side and will travel with it. Unfortunately I need to do a little manual layout for that.

Comment (jasonLaster) [inline public/js/components/Editor.js]: This code seems like it should live in `util/editor` as it is CM specific and does not deal with the business logic of debugging per-se

File: public/js/components/EditorBreakpoint.js [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
+  svg.setAttribute("preserveAspectRatio", "none");
+  let path = document.createElementNS("http://www.w3.org/2000/svg", "path");
+  // Until we figure out our loader story, embed it directly so we can
+  // control it with CSS.
+  path.setAttribute("d", "M27.1,0H1C0.4,0,0,0.4,0,1v10c0,0.6,0.4,1,1,1h26.1 c0.6,0,1.2-0.3,1.5-0.7L33,6l-4.4-5.3C28.2,0.3,27.7,0,27.1,0z"); // eslint-disable-line max-len
+  svg.appendChild(path);
+  marker.appendChild(svg);
+
   return marker;
 }
```
Comment (jasonLaster) [inline public/js/components/EditorBreakpoint.js]: nice

File: public/js/components/Editor.js (hunk 1 above)
Comment (jlongster) [inline public/js/components/Editor.js]: That's fine with me

**Comment by jlongster** [comment]
I asked Helen to look at this too, so let's not merge until she's had a look

**Comment by helenvholmes** [comment]
Here's a huge debugger breakpoint (hopefully this is correct): http://cl.ly/0V392o061S2O

Here's a screenshot of the issue at very large files: http://cl.ly/3S0q1a3G3P2X

bgrins' test page: http://bgrins.github.io/devtools-demos/debugger/scripts.html

**Comment by jlongster** [comment]
Fixed the wide breakpoints, and also fixed the mis-aligned line on my computer. I didn't have "SF UI Text" installed and the line number gutter was trying to use that, but it should be using the monospaced font. Everything looks so much better now!!

**Pull request merged**